CHANGELOG.md

```diff
@@ -226,6 +226,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Unify checkout identifier in checkout mutations and queries - #7511 by @IKarbowiak
 - Use root level channel argument for filtering and sorting - #7374 by @IKarbowiak
   - drop `channel` field from filters and sorters
+- Drop top-level `checkoutLine` query from the schema with related resolver - #3832 by @dexon44
```

Comment:
It's good to mention what to use instead, search `instead` in this file for example.

Reply:
In this case, we don't have a 1:1 replacement, but we can say that the `checkout` query should be used instead. We want people to get the checkout, and from there fetch the lines.

Reply:
CC @dexon44 ⬆️